WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -66,6 +66,10 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
     @Published private(set) var selectedPredefinedPackage: ShippingLabelPredefinedPackage?
     @Published var totalWeight: String
 
+    /// Whether the user has edited the total package weight. If true, we won't make any automatic changes to the total weight.
+    ///
+    @Published var isPackageWeightEdited: Bool = false
```

Comment:
Since we're not observing `isPackageWeightEdited`, I think it's safe to remove `@Published` here and keep it a plain variable. What do you think?